test_runner/.gitignore

```diff
@@ -7,3 +7,4 @@ xctestrun/
 src/test/kotlin/ftl/fixtures/error_result/*.xml
 src/test/kotlin/ftl/fixtures/success_result/*.xml
 *.json
+*-describe.adoc
```

Comment:
Why is describe ignored?

Reply:
describe.adocs is a middle file for Piccoli doc generation, it contains some metadata like
```
// end::picocli-generated-man-section-synopsis[]

// tag::picocli-generated-man-section-description[]
```

Reply:
makes sense